store.py

```diff
@@ -182,31 +361,45 @@ class CheckoutLine:
     is_open: bool
     queue: List[Customer]
 
-    def __init__(self, capacity: int) -> None:
+    def __init__(self, capacity: int, is_open: True, queue: []) -> None:
         """Initialize an open and empty CheckoutLine.
 
-        >>> line = CheckoutLine(1)
+        >>> line = CheckoutLine(1, True, [])
         >>> line.capacity
         1
         >>> line.is_open
         True
         >>> line.queue
         []
         """
+        self.capacity = capacity
+        self.is_open = is_open
+        self.queue = queue
 
     def __len__(self) -> int:
         """Return the size of this CheckoutLine.
+        >>> line = CheckoutLine(10, True, [Customer('bill', []), \
```

Comment:
considering that signature of the method shall not be changed, this is incorrect